spring-cloud-gateway-server/src/main/java/org/springframework/cloud/gateway/config/HttpClientFactory.java

```diff
@@ -127,43 +110,6 @@ private HttpClient applyCustomizers(HttpClient httpClient) {
 		return httpClient;
 	}
 
-	protected HttpClient configureSsl(HttpClient httpClient) {
```

Comment:
We might need to chat about this. Again, since this is a public API, these can't be arbitrarily deleted.

Reply:
This can easily be restored, just calling the new component 
```java
	protected HttpClient configureSsl(HttpClient httpClient) {
		return sslConfigurer.configureSsl(httpClient);
	}
```